core/frontend/services/themes/storage.js

```diff
@@ -69,17 +69,19 @@ module.exports = {
                 return themeLoader.loadOneTheme(shortName);
             })
             .then((loadedTheme) => {
+                let themeOverriden = false;
```

Comment:
Maybe doing something like:
```
const overrideTheme = (shortName === settingsCache.get('active_theme'));

if (overrideTheme) {
...
}

return {
  themeOverridden: overrideTheme
  ...
}
```

Would give it a better context?

Also, notice `Overridden` - `dd` double `d` ;)

Reply:
😄 That's nicer, updated now.